src/meta/src/hummock/compaction/level_selector.rs

```diff
@@ -393,9 +392,10 @@ impl LevelSelector for DynamicLevelSelector {
                     self.task_type(),
                 ));
             }
-            selector_stats
-                .skip_picker
-                .push((select_level, target_level, stats));
+
+            for local_picker_stat in local_picker_stats {
+                selector_stats.skip_picker.push(local_picker_stat);
+            }
         }
```

Comment:
nits: How about passing `LocalSelectorStatistic` to `pick_compaction` instead of leting `pick_compaction` return Vec<LocalPickerStatistic> and then pushing them into `LocalSelectorStatistic`? In this way, we can unify the interface of `pick_compaction` for all types of picker and reduce unnecessary code changes in tests.